frontend/src/scenes/funnels/funnelLogic.ts

```diff
@@ -127,10 +160,9 @@ export const funnelLogic = kea<funnelLogicType<TimeStepOption>>({
                             ? { breakdown: null, breakdown_type: null }
                             : {}),
                     }
-                    let result: FunnelResult
+                    let result // Tricky: pollFunnel would ideally return a FunnelResult or FunnelResult<FunnelStep[][]> based on whether its input params include breakdown
```

Comment:
Could you type it accordingly then? Removing a type, and adding `any` with a 🤷‍♂️ comment is not really the solution :).